src/core/utils/helpers/general.ts

```diff
@@ -75,7 +75,8 @@ export const getFirstPublishedManifestation = (
 
 export const getFirstPublishedYear = (manifestations: Manifestation[]) => {
   return String(
-    getFirstPublishedManifestation(manifestations)?.publicationYear.display
+    getFirstPublishedManifestation(manifestations)?.edition?.publicationYear
```

Comment:
I think we could make a function called getManifestationPublicationYear that receives a manifestation and returns: `string | null`
So: 

```ts
getManifestationPublicationYear(manifestation: Manifestation): string | null
```
I am not fan og the long optional chaining and by having it done in one place, we only need to change that function if/when it changes again.